Patch to support the versions of rand and rand-chacha in Debian
based on a revert of upstream commit 0872f98c151892b3b417eccbf22bc023a30433fb

Index: blake3/src/test.rs
===================================================================
--- blake3.orig/src/test.rs
+++ blake3/src/test.rs
@@ -429,7 +429,7 @@ fn test_fuzz_hasher() {
         let mut total_input = 0;
         // For each test, write 3 inputs of random length.
         for _ in 0..3 {
-            let input_len = rng.gen_range(0..(INPUT_MAX + 1));
+            let input_len = rng.gen_range(0, INPUT_MAX + 1);
             #[cfg(feature = "std")]
             dbg!(input_len);
             let input = &input_buf[total_input..][..input_len];
Index: blake3/Cargo.toml
===================================================================
--- blake3.orig/Cargo.toml
+++ blake3/Cargo.toml
@@ -49,10 +49,10 @@ version = "0.4.2"
 version = "0.12.0"
 
 [dev-dependencies.rand]
-version = "0.8.0"
+version = "0.7.2"
 
 [dev-dependencies.rand_chacha]
-version = "0.3.0"
+version = "0.2.1"
 [build-dependencies.cc]
 version = "1.0.4"
 
